packages/nextjs/app/builders/0x23103d42Bc69C55617Ce265671Dea2907073ede8/page.tsx

```diff
@@ -0,0 +1,94 @@
+          <div className="card-body p-8 md:p-12">
+            {/* Header Section */}
+            <div className="text-center mb-8">
+              <div className="avatar mb-4">
+                <div className="w-24 h-24 rounded-full ring-4 ring-accent ring-offset-base-100 ring-offset-4">
+                  <Image
+                    src="/tony.png"
+                    alt="Tonny avatar"
+                    width={96}
+                    height={96}
+                    className="w-full h-full object-cover rounded-full"
+                  />
+                </div>
+              </div>
+              <h1 className="text-4xl md:text-5xl font-bold mb-2 bg-gradient-to-r from-blue-400 via-purple-400 to-pink-400 bg-clip-text text-transparent">
+                Tonny
+              </h1>
+              <div className="flex justify-center mb-4">
+                <Address address="0x23103d42Bc69C55617Ce265671Dea2907073ede8" />
+              </div>
+            </div>
+
+            {/* Bio Section */}
+            <div className="text-center mb-8">
+              <div className="p-6 rounded-lg bg-gradient-to-br from-base-200 to-base-300/50 border border-accent/10">
+                <h3 className="text-lg font-semibold mb-4 text-base-content/90">About Me</h3>
+                <p className="text-base md:text-lg text-base-content/80 leading-relaxed">
+                  Aspiring blockchain developer on a journey to master Web3 technologies. Passionate about building
+                  decentralized applications and exploring the potential of smart contracts. Currently diving deep into
+                  Solidity and expanding my skills in the DeFi ecosystem.
+                </p>
+                <div className="mt-4 flex justify-center gap-2 flex-wrap">
+                  <span className="badge badge-accent">Solidity</span>
+                  <span className="badge badge-info">Web3</span>
+                  <span className="badge badge-success">Smart Contracts</span>
+                  <span className="badge badge-warning">DeFi</span>
+                  <span className="badge badge-secondary">React</span>
+                  <span className="badge badge-primary">TypeScript</span>
+                </div>
+              </div>
+            </div>
+
+            {/* Social Links */}
+            <div className="mb-8">
+              <h2 className="text-2xl font-bold mb-4 text-center">Connect</h2>
+              <div className="flex flex-wrap gap-3 justify-center">
+                {SOCIAL_LINKS.map(({ href, label, icon }) => (
+                  <Link
+                    key={href}
+                    href={href}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="btn btn-outline btn-accent gap-2 hover:scale-105 hover:shadow-lg transition-all"
+                  >
+                    <span>{icon}</span>
+                    {label}
+                  </Link>
+                ))}
```

Comment:
There's only one SOCIAL_LINKS, maybe you shouldn't be iterating.

Also: This button has no contrast against the page background, consider inverting it's color.